staging/src/k8s.io/apiserver/pkg/server/filters/timeout_test.go

```diff
@@ -243,6 +266,10 @@ func TestTimeoutHeaders(t *testing.T) {
 	if err != nil {
 		t.Fatal(err)
 	}
+
+	// wait for the request handler to complete
+	<-handlerDoneCh
+
```

Comment:
I don't think waiting for the handler to be done here is doing much, can drop this